src/operations/command.ts

```diff
@@ -63,6 +64,13 @@ export interface CommandOperationOptions
    * This option is deprecated and will be removed in an upcoming major version.
    */
   noResponse?: boolean;
+
+  /**
+   * Used when the command needs to grant access to the underlying namespaces for time series collections.
+   * Only available on server versions 8.2 and above.
```

Comment:
We use the `@sinceServerVersion` tag when there is a server version requirement on options.  Can we add that here as well?